tracker/view/add.py

```diff
@@ -199,4 +200,22 @@ def add_group():
 
     db.session.commit()
     flash('Added {}'.format(group.name))
+
+    missing_lib32_variant(pkgnames, group)
+
     return redirect('/{}'.format(group.name))
+
+
+def missing_lib32_variant(pkgnames, group):
+    for pkgname in pkgnames:
```

Comment:
I tested this locally and the behavior is correct. I wanted to make sure that we don't add a "there's a lib32 package added but this still flashes the warning" behavior. Could we add a test for it?

Reply:
Good idea!